torchvision/transforms/v2/functional/_geometry.py

```diff
@@ -1548,7 +1548,7 @@ def _perspective_image_pil(
     image: PIL.Image.Image,
     startpoints: Optional[List[List[int]]],
     endpoints: Optional[List[List[int]]],
-    interpolation: Union[InterpolationMode, int] = InterpolationMode.BICUBIC,
+    interpolation: Union[InterpolationMode, int] = InterpolationMode.BILINEAR,
```

Comment:
We have used bicubic here, but bilinear for the tensor kernels, since that is what v1 does:

https://github.com/pytorch/vision/blob/e13b8f5c3616bdc58fa847a848d63acdd416a692/torchvision/transforms/_functional_pil.py#L321

https://github.com/pytorch/vision/blob/e13b8f5c3616bdc58fa847a848d63acdd416a692/torchvision/transforms/_functional_tensor.py#L704

However, since the PIL kernel is private in v1, this divergence doesn't really matter, since they are always called through the functional

https://github.com/pytorch/vision/blob/e13b8f5c3616bdc58fa847a848d63acdd416a692/torchvision/transforms/functional.py#L719

https://github.com/pytorch/vision/blob/e13b8f5c3616bdc58fa847a848d63acdd416a692/torchvision/transforms/functional.py#L758-L762

Although in v2 the PIL kernel is private as well I see no reason for this divergence. LMK if you think otherwise.

Reply:
Good catch, thanks

Reply:
Sooooo this needs a cherry-pick?

Reply:
Not really. PIL kernel is private for both v1 and v2. And thus their default values do not matter, but only the default values of the functional that calls them, since that overwrites the default. And that is bilinear and thus aligned with the public tensor kernels.

Reply:
oh right it's private, all good then. Thanks!